### DISCUSSION TITLE
Scrollbar color?

### DISCUSSION THREAD

**Discussion by NailGit** [ahrm/sioyek#818] [General]
Is it possible to change the scrollbar color? In my case, I have a white scrollbar and a white scroll-marker. It is very difficult to see a white scroll-marker on the white scrollbar.

**Comment by ahrm** [comment]
Not currently, we don't customize the scrollbar, so it should be the default Qt scrollbar.

**Reply by NailGit** [reply]
Ok. Maybe in the future. In any case, you made a very useful tool for normal and effective working with pdf files. For this reason, I used many years mupdf. You made more tools very usefull. 

Maybe it is simpler to realize the configuration of Qt in a separate file css, as I use in TemStudio and Mathematica? For example, Mathematica uses a separate file frontend.css in which I put info about scrollbar:
`QScrollBar:vertical{background: #b7b398; width: 12px;  margin: 0px 0px 0px 0px;}

QScrollBar::handle:vertical{background: #1E90FF; border: 1px solid transparent; border-radius: 5px; border-color: beige;  min-height: 60px;  width: 12px; }`

As a result, I have a blue scroll-marker, is very visible.

**Reply by NailGit** [reply]
I compiled Sioyek and now the scroll bar is configured as qt5 in my system. The same scroll as in TexSudio and Mathematica. Looks well.
